prac_08/taxi.py

```diff
@@ -7,11 +7,11 @@
 
 class Taxi(Car):
     """Specialised version of a Car that includes fare costs."""
+    price_per_km = 1.23
 
-    def __init__(self, name, fuel, price_per_km):
```

Comment:
Since price_per_km is already defined in line 10, including it in the init does not do anything.

Reply:
Resolved in future commit. Thank you